recipes/4ti2/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "4ti2" %}
+{% set version = "1.6.9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/4ti2/4ti2/releases/download/Release_{{ version.replace(".", "_") }}/4ti2-{{ version }}.tar.gz
+  sha256: 3053e7467b5585ad852f6a56e78e28352653943e7249ad5e5174d4744d174966
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - automake
+    - libtool
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - gmp
+    - glpk
+  run:
+    # We link against glpk which does not specify a run_exports so we need to add it explicitly here.
+    - glpk
```

```suggestion
    - {{ pin_compatible("glpk", max_pin="x.x") }}
```

Can you send a PR for glpk to add run_exports?

---

Also, can you send a PR to `conda-forge-pinning-feedstock`?

---

There's a pin in conda-forge-pinning-feedstock. So I don't think this is necessary.